.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release Charts
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'chart/**'
+
+jobs:
+  release:
+    permissions:
+      contents: write # needed to write releases
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3
```

Comment:
May I ask why are you using commit hash for the action instead of the version, like:

```yaml
  - uses: actions/checkout@v3
```